packages/fxa-auth-server/lib/metrics/amplitude.js

```diff
@@ -38,6 +38,7 @@ const EMAIL_TYPES = {
   recoveryEmail: 'reset_password',
   unblockCode: 'unblock',
   verifyEmail: 'registration',
+  verifyShortCodeEmail: 'registration',
```

Comment:
This was needed for amplitude to log the email metrics, it was in the documentation I just missed it 🤦🏽‍♂️

Reply:
I wonder if we can prevent this in future by having a check when the auth-server starts up, that checks whether all the known emails have a corresponding entry in this list.

Reply:
I agree, having a check would have helped catch this 👍🏽, we could probably/add extend this in this [issue](https://github.com/mozilla/fxa/issues/511).